src/components/CreateMenu/BaseCreateMenu.js

```diff
@@ -35,6 +35,15 @@ class BaseCreateMenu extends PureComponent {
                 animationOut={this.props.animationOut}
             >
                 <View style={this.props.isSmallScreenWidth ? {} : styles.createMenuContainer}>
+                    {this.props.headerText && (
```

Comment:
I couldn't find the definition of this prop, I don't see it in `CreateMenu`, `BaseCreateMenu`, or `createMenuPropTypes`. Is this missing?